Marlin/src/module/planner.h

```diff
@@ -260,6 +260,10 @@ typedef struct PlannerBlock {
     uint8_t  la_scaling;                    // Scale ISR frequency down and step frequency up by 2 ^ la_scaling
     uint16_t max_adv_steps,                 // Max advance steps to get cruising speed pressure
              final_adv_steps;               // Advance steps for exit speed pressure
+    #if ENABLED(LA_ZERO_SLOWDOWN)
+      uint32_t current_la_step_rate;        // Current (gradually changing) linear advance rate
```

Comment:
I think `current_la_step_rate` does not need to be in PlannerBlock as it only applies to the currently executing block in the stepper module. Move it to `stepper.h` to save a bunch of RAM and also remove a bunch of indirection overhead.